### ISSUE TITLE
Azure Queue Sink does not turn milliseconds into seconds like source does

### ISSUE THREAD

**Issue by cross311** [walmartlabs/cookie-cutter#85]
Azure Queue expects the visibility timeout to be in seconds... however most things in cookie cutter take in milliseconds.  So in the source we turn milliseconds from the constructor and turn in the seconds.  However in the sink where we actually set the visibility timeout when creating a new message, we pass it straight in.

REQUEST: please take in Milliseconds like everywhere else and turn it in to seconds to hide the underlying library from consumers.

**Comment by sklose** [comment]
makes sense, we should probably add new metadata fields then to ensure this remains backwards compatible.

**Comment by plameniv** [comment]
Added `VisibilityTimeoutMs` and `TimeToLiveMs` fields to the metadata for use with calls to publish and store (when directly setting the metadata fields). Will keep the old ones for compatibility.

**Comment by plameniv** [comment]
Reopening, since the main concern hasn't been addressed.

We fixed a breaking API change to Azure Queues. The config passed to the Queue constructor is back to expecting seconds.
